docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
just as a note for future improvements: we should also really document how to build it via Docker.